lib/jekyll_auth/config.rb

```diff
@@ -3,21 +3,36 @@ def self.config_file
     File.join(Dir.pwd, '_config.yml')
   end
 
+  def self.jekyll_config
+    @config ||= begin
+      jekyll_config = YAML.safe_load_file(config_file)
+    rescue
+      {}
+    end
+  end
+
+  def self.destination
```

Comment:
Don't know that we need to memoize, since this is just pulling a value by key from a hash, which is cheap, and same here regarding memoization.